docs/Installation/Installation_Guide.rst

```diff
@@ -1,579 +1,581 @@
+    kube-ingress   nginx-ingress-default-backend-6664bc64c9-xsws5         1/1       Running     0          1m
+    kube-ops       alertmanager-kube-prometheus-0                         2/2       Running     0          2m
+    kube-ops       alertmanager-kube-prometheus-1                         2/2       Running     0          2m
+    kube-ops       es-client-7cf569f5d8-2z974                             1/1       Running     0          2m
+    kube-ops       es-client-7cf569f5d8-qq4h2                             1/1       Running     0          2m
+    kube-ops       es-data-cd5446fff-pkmhn                                1/1       Running     0          2m
+    kube-ops       es-data-cd5446fff-zzd2h                                1/1       Running     0          2m
+    kube-ops       es-exporter-elasticsearch-exporter-7df5bcf58b-k9fdd    1/1       Running     3          1m
+
+    [...]
+
+    kube-system    kubernetes-dashboard-b795f77cd-qncpl                   1/1       Running     0          2m
+    kube-system    metrics-server-5b59ccccfd-4brrz                        1/1       Running     0          2m
+    kube-system    tiller-deploy-5c688d5f9b-ffzsg                         1/1       Running     0          2m
+
+* Or list all deployed Helm_ applications::
+
+    (metal-k8s) $ helm list
+    NAME                  REVISION  UPDATED                   STATUS    CHART                         NAMESPACE
+    cerebro               1         Tue Jul 24 22:52:18 2018  DEPLOYED  cerebro-0.3.0                 kube-ops
+    elasticsearch         1         Tue Jul 24 22:51:23 2018  DEPLOYED  elasticsearch-1.3.0           kube-ops
+    elasticsearch-curator 1         Tue Jul 24 22:51:32 2018  DEPLOYED  elasticsearch-curator-0.3.0   kube-ops
+    es-exporter           1         Tue Jul 24 22:52:07 2018  DEPLOYED  elasticsearch-exporter-0.2.0  kube-ops
+    fluent-bit            1         Tue Jul 24 22:51:50 2018  DEPLOYED  fluent-bit-0.6.0              kube-ops
+    fluentd               1         Tue Jul 24 22:51:41 2018  DEPLOYED  fluentd-0.1.4                 kube-ops
+    heapster              1         Tue Jul 24 22:50:58 2018  DEPLOYED  heapster-0.3.0                kube-system
+    kibana                1         Tue Jul 24 22:51:59 2018  DEPLOYED  kibana-0.8.0                  kube-ops
+    kube-prometheus       1         Tue Jul 24 22:50:45 2018  DEPLOYED  kube-prometheus-0.0.96        kube-ops
+    nginx-ingress         1         Tue Jul 24 22:49:30 2018  DEPLOYED  nginx-ingress-0.23.0          kube-ingress
+    prometheus-operator   1         Tue Jul 24 22:50:03 2018  DEPLOYED  prometheus-operator-0.0.27    kube-ops
+
+.. _Helm: https://www.helm.sh
+
+Cluster Services
+----------------
+
+Services to operate and monitor your MetalK8s cluster are provided. To access
+these dashboards:
+
+1. Make sure kubectl is installed on your local machine::
+
+   $ yum install kubectl

```

Comment:
You get this in a `make shell` environment, which also ensures you get a version which works with the deployed cluster, which is definitely not guaranteed when installing from CentOS repositories.

Reply:
deleted.